opengrok-tools/src/test/python/test_commandSequence.py

```diff
@@ -77,6 +77,20 @@ def test_exit_2_handling():
     assert not cmds.failed
 
 
+@pytest.mark.skipif(not os.path.exists('/bin/sh') or not os.path.exists('/bin/echo'), reason="requires Unix")
+def test_driveon_flag():
+    cmds = CommandSequence(CommandSequenceBase("opengrok-master",
+                                               [{"command": ["/bin/sh", "-c",
+                                                             "echo " + CommandSequence.PROJECT_SUBST + "; exit 2"]},
+                                                {"command": ["/bin/echo"]}], driveon=True))
```

Comment:
I'd vote for `skip_failed` or similar

Reply:
you mean rename `driveon` ? The skip failed would not be precise because in fact the cycle is terminated for failed commands (i.e. those that return non-zero but also non-2) if this option is True.

Reply:
I don't see this in the code, I see that every error is ignored and the cycle is never terminated when this option = True that's why I'd be for the other name.

Reply:
Ah, logic error. Should be fixed now.

Reply:
ok